File: fizzbuzz/fizzbuzz.exs
defmodule Operator do
  def eval(0), do: [0]
  def eval(elem) when rem(elem,5)==0 and rem(elem,3)==0, do: eval(elem-1) ++[:fizzbuzz]
  def eval(elem) when rem(elem,3)==0, do: eval(elem-1) ++ [:fizz]
  def eval(elem) when rem(elem,5)==0, do: eval(elem-1) ++ [:buzz]
  def eval(elem), do: eval(elem-1) ++ [elem]
end

File: fizzbuzz/fizzbuzz.py
print ("FizzBuzz implementation with python")
count = 0
while count < 15:
    count = count + 1
    if count%3==0 and count%5==0:
        print ("FizzBuzz")
    elif count%3==0:
        print ("Fizz")
    elif count%5==0:
        print ("Buzz")
    else:
        print (count)
